### PR TITLE
Bam file testing

### PR DESCRIPTION
**Pull request by ghost**
- the bam file testing was added
- the data files are moved to a new folder
- lots of autopep8 linting happening
- commenting out of files that would fail on tests
- fixed the relative paths

### CODE REVIEW

**Comment by jkanche** [comment]
the only change in the bigwig and bigbed files should be related to updating file paths (since all test files are moved to `tests/data`)

**Comment by ghost** [comment]
bigwig and bigbed files had linting done to them from autopep 8

File: src/epivizFileParser/BamFile.py [hunk 1]
```diff
@@ -50,10 +52,11 @@ def get_col_names(self, result):
         Columns of a bam file
         """
         if self.columns is None:
-            self.columns = ["chr", "start", "end", "number of sequence aligned"]
+            self.columns = ["chr", "start", "end",
+                            "number of sequence aligned"]
```
Comment (jkanche) [inline src/epivizFileParser/BamFile.py:56]: lets call this value

Comment (ghost) [reply src/epivizFileParser/BamFile.py:56]: columns?

also that was linter doing it's things

merged already...

**Pull request merged**